### ISSUE TITLE
Acute retinal necrosis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10984]
Preferred term label:
Acute retinal necrosis

Synonyms
Acute necrosis of the retina

Definition (free text, please give PubMed ID)
A condition characterized by the death of retinal tissue due to ischemia, infection, or inflammation. Retinal necrosis can result from vascular occlusion, viral infections such as varicella-zoster virus (VZV) or cytomegalovirus (CMV), or autoimmune processes. Clinically, it presents with retinal whitening, hemorrhages, and potential retinal detachment, often leading to severe visual impairment or blindness if untreated. 

Parent term (use hpo.jax.org/app)
Abnormal retinal morphology (HP:0000479)

Diseases characterized by this term
Acute retinal necrosis, Cytomegalovirus retinitis

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
See also https://pubmed.ncbi.nlm.nih.gov/29262034/
Is this a phenotypic feature or an inference? To be discussed.

**Comment by daboe01** [comment]
Treating ARN as a specific phenotypic feature aligns better with established practice (see e.g. Keratoconus) and provides a more precise and useful description of the observable clinical condition than considering it merely an inference or trying to represent it solely through its constituent parts (which would be hard given the terms we have).

**Comment by pnrobinson** [comment]
@daboe01  -- THANKS!
Adding new term: 
- ID: HP:6001370
- label: Acute retinal necrosis
- parents: [Abnormal retinal morphology (HP:0000479)](https://hpo.jax.org/app/browse/term/HP:0000479)
- pmid: PMID:29262034
- def: A condition characterized by the death of retinal tissue due to ischemia, infection, or inflammation.
- comment: Retinal necrosis can result from vascular occlusion, viral infections such as varicella-zoster virus (VZV) or cytomegalovirus (CMV), or autoimmune processes. Clinically, acute retinal necrosis presents with retinal whitening, hemorrhages, and potential retinal detachment, often leading to severe visual impairment or blindness if untreated.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10984
- synonyms: Acute necrosis of the retina(NONE)
